wheel/src/api.rs

```diff
@@ -89,11 +90,22 @@ pub fn get_puzzle_and_solution_for_coin<'py>(
     find_parent: Bytes32,
     find_amount: u64,
     find_ph: Bytes32,
+    flags: u32,
 ) -> PyResult<(&'py PyBytes, &'py PyBytes)> {
     let mut allocator = make_allocator(LIMIT_HEAP);
-    let program = node_from_bytes(&mut allocator, program)?;
-    let args = node_from_bytes(&mut allocator, args)?;
-    let dialect = &ChiaDialect::new(LIMIT_STACK);
+
+    let (program, args) = if (flags & ALLOW_BACKREFS) != 0 {
```

Comment:
A little petty but slightly shorter.

```suggestion
    let nfb = if (flags & ALLOW_BACKREFS) != 0 { node_from_bytes_backrefs } else { node_from_bytes };
    let program = nfb(&mut allocator, program)?;
    let args = nfb(&mut allocator, args)?;

```